.github/workflows/build.yml

```
@@ -1,4 +1,4 @@
-name: build
+name: osxbuild
```

I think the term `osx` is outdated, let's use macOS:
```suggestion
name: macosbuild
```